services/topology-engine/Dockerfile

```diff
@@ -14,23 +14,12 @@
 #
 
 FROM kilda/base-ubuntu
-RUN apt-get update
-RUN apt-get install -y python-pip
-RUN apt-get install -y supervisor
-RUN apt-get install -y nginx
-RUN apt-get install -y vim
-RUN pip install --upgrade pip
-RUN pip install neo4j-driver
-RUN pip install kafka
-RUN pip install uwsgi
-RUN pip install pyotp
-RUN pip install jsonschema
-RUN pip install py2neo
-RUN pip install requests
-RUN pip install gevent
-RUN pip install kazoo
-RUN pip install python-logstash
-RUN chmod +x /usr/local/bin/uwsgi
+RUN apt-get update && \
+      apt-get install -y python-pip supervisor vim && \
+      pip install --upgrade pip && \
+      pip install neo4j-driver \
+      kafka py2neo gevent python-logstash && \
```

Comment:
This line ends in && \  ... but there is no next line

Reply:
Fixed